{{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/apps/contact/models.py

```diff
@@ -0,0 +1,110 @@
+from cms.apps.pages.models import ContentBase
+from cms.models import HtmlField
+from django.db import models
+
+
+class Contact(ContentBase):
+
```

Comment:
You'll probably want to add a content type icon here. See https://github.com/onespacemedia/cms-icons/